### ISSUE TITLE
'create_key' fails: AttributeError: 'str' object has no attribute 'items'

### ISSUE THREAD

**Issue by AlekseiMarinichenko** [JWCook/aiohttp-client-cache#86]
### The problem
**aiohttp-client-cache** fails when I pass JSON to the `data` argument via POST request.  Without **aiohttp-client-cache** the code works fine.

```
../../../backend/src/apis/api.py:496: in _request
    async with self.session.request(method=method, url=url, params=params, headers=headers, **kwargs) as response:
../../../venv/lib/python3.9/site-packages/aiohttp/client.py:1117: in __aenter__
    self._resp = await self._coro
../../../venv/lib/python3.9/site-packages/forge/_revision.py:322: in inner
    return await callable(*mapped.args, **mapped.kwargs)
../../../venv/lib/python3.9/site-packages/aiohttp_client_cache/session.py:28: in _request
    response, actions = await self.cache.request(method=method, url=str_or_url, **kwargs)
../../../venv/lib/python3.9/site-packages/aiohttp_client_cache/backends/base.py:105: in request
    key = self.create_key(method, url, **kwargs)
../../../venv/lib/python3.9/site-packages/aiohttp_client_cache/backends/base.py:205: in create_key
    return create_key(
../../../venv/lib/python3.9/site-packages/aiohttp_client_cache/cache_keys.py:34: in create_key
    key.update(encode_dict(data))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

data = '{"data": {"passengers": [{"type": "adult"}, {"age": 30}], "slices": [{"origin": "SFO", "destination": "JFK", "departure_date": "2021-09-02"}], "cabin_class": "economy"}}'

    def encode_dict(data):
>       item_pairs = [f'{k}={v}' for k, v in sorted((data or {}).items())]
E       AttributeError: 'str' object has no attribute 'items'

../../../venv/lib/python3.9/site-packages/aiohttp_client_cache/cache_keys.py:63: AttributeError
```

### Expected behavior
The code words without `aiohttp-client-cache`, so `aiohttp-client-cache` must support the same.

### Steps to reproduce the behavior
Here is a part of my code:
```python
method = "POST"
url = ...  # I think it does not matter
params = None
headers = {
            "Accept-Encoding": "gzip",
            "Accept": "application/json",
            "Content-Type": "application/json",
            "Service-Version": "beta",
            "Authorization": f"Bearer {API_KEY}",
        }
data = {
            "data": {
                # A dictionary here, it does not matter here
        }
data=json.dumps(data)
async with self.session.request(method=method, url=url, params=params, headers=headers) as response:
    ...
```

### Workarounds

I do not know.

### Environment
- aiohttp-client-cache version: 0.4.3
- Python version: 3.9
- Platform: Lubuntu 20.04

**Comment by JWCook** [comment]
Thanks for reporting this! This should be easy to fix.

**Comment by JWCook** [comment]
This is now fixed in the latest pre-release build (`0.5.0.dev1`). Give that a try, any let me know if you have any more issues.

**Comment by AlekseiMarinichenko** [comment]
Great, now it works with the latest version (`pip install git+https://github.com/JWCook/aiohttp-client-cache`). Thank you. I think that I can close the issue.

I will test the latest version again my project's all tests later.
